X16 Reference - 09 - VERA Programmer's Reference.md

```diff
@@ -931,16 +934,29 @@ The **Left** and **Right** bits control to which output channels the sound shoul
 | 2        | Triangle    |
 | 3        | Noise       |
 
-**Pulse width** controls the duty cycle of the pulse waveform. A value of 63 will give a 50% duty cycle or square wave, 0 will give a very narrow pulse.
+**Pulse Width / XOR** controls the duty cycle of the pulse waveform or the XOR 
+permutation when used with the triangle or saw. For pulse, a value of 63 ($3F) will
+give a 50% duty cycle or square wave, 0 will give a very narrow pulse.
+
+When the triangle or saw waveform is selected, the value influences an XOR calculation
+the changes the resulting waveform. This is most noticeable with the triangle waveform. 
+It can be used to provide an NES-like fuzzy triangle as well as an overdriven saw sound
+(similar to the VRC6 NES chip) among several other varieties of sounds.
+
+When used with the saw, the result is more substle. It adds some overtones to 
+the saw.
 
-Just like the other waveform types, the frequency of the noise waveform can be controlled using frequency. In this case a higher frequency will give brighter noise and a lower value will give darker noise.
+Setting the PW/XOR to 00 for Tri/Saw inverts the waveform from what it was prior
+to the addition of the XOR feature. Be careful of phasing as a result.
 
+**Noise** Just like the other waveform types, the frequency of the noise waveform can be controlled using frequency. In this case a higher frequency will give brighter noise and a lower value will give darker noise. The PWM/XOR values do not influence
+the noise shape.
 
-## PCM audio
+### PCM audio
 
 For PCM playback, VERA contains a 4kB FIFO buffer. This buffer needs to be filled in a timely fashion by the CPU. To facilitate this an **AFLOW** (Audio FIFO low) interrupt can be generated when the FIFO is less than 1/4 filled.
 
-### Audio registers
+#### Audio registers
```

Comment:
Heading hierarchy seems a bit off. If "Audio registers" is made h4, the respective register headings should be h5, and "Audio data formats" also h4 (since they also relate to PCM).

That does feel like some very deep nesting into the hierarchy, but I think that's a symptom of most of the VERA stuff being on the one page, an issue for another day.